src/store/reducers/UserSlice.ts

```diff
@@ -79,9 +79,19 @@ export const counterSlice = createSlice({
       (action.payload === Сondition.active || Сondition.completed || Сondition.all) &&
         (state.filtValue = action.payload);
     },
+    sortTodosBy: (state, action: PayloadAction<string>) => {
+      SortOptions.message === action.payload &&
+        (state.todos = state.todos.sort((a: ITodo, b: ITodo) => (a.message > b.message ? 1 : -1)));
```

Comment:
Compare your messages using localeCompare

Reply:
Added

Reply:
I rewrite in slice sort, but think about need or not to rewrite with localeCompare in all project ?

Reply:
Because I see in some moments it looks more complicated

Reply:
Added changes to the whole project